sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -1506,6 +1506,52 @@ public async Task OpenReadAsync_Seek(long offset)
             TestHelper.AssertSequenceEqual(expectedData, outputStream.ToArray());
         }
 
+        [Test]
+        // lower position within _buffer
+        [TestCase(400)]
+        // higher positiuon within _buffer
+        [TestCase(500)]
+        // lower position below _buffer
+        [TestCase(250)]
+        // higher position above _buffer
+        [TestCase(550)]
```

Comment:
Should we add test cases for what happens if we go out of bound ?

Reply:
Will do before I merge.

Reply:
Actually we already have Seek() tests for these cases, and we are just calling into Seek().  I don't think it is necessary.